hstrat/test_drive/descend_template_phylogeny_/_descend_template_phylogeny_posthoc.py

```diff
@@ -99,10 +163,10 @@ def descend_template_phylogeny_posthoc(
     descending_tree_iterable: typing.Iterable,
     get_parent: typing.Callable[[typing.Any], typing.Any],
     get_stem_length: typing.Callable[[typing.Any], int],
-    seed_column: HereditaryStratigraphicColumn,
```

Comment:
We should probably keep `seed_column` but throw a deprecation warning if it's used.